### PR TITLE
interactive bots: Improve task descriptions

### PR DESCRIPTION
**Pull request by KaiCode2**
When I was doing this task the wording "[Replace] placeholder `<email>` with the user email" was a bit unclear as to wether it was referring to the current user's email or the bot's.

Included are a couple of other minor grammar tweaks to make it read a bit more smoothly.

Let me know what you all think of the wording and if this merits the rewording proposed.

Task can be found [here.](https://codein.withgoogle.com/dashboard/task-instances/5675555963797504/)

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@KaiCode2, it looks like you've already signed the Dropbox CLA. Thanks!

**Comment by reyha** [comment]
Hi @KaiCode2 Your changes look good. However, you have to rename your PR to ``interactive bots: Improve task descriptions`` as per the task description. Also,I think it would be best if you could find any other typos/confusions that you can add to your commit.

**Comment by KaiCode2** [comment]
Hello @reyha I added a couple more changes that I found a bit confusing in task A. I also threw in a couple links to mentioned pages. Let me know what you think.

File: tasks/interactive-bots.md [hunk 1]
```diff
@@ -59,14 +59,14 @@ Follow the tutorial below to test run the followup bot on your Zulip instance
   site=<dev-url>
   ```
 
-  Replace the placeholder `<api-key>` in the example with an existing user API
-  key and placeholder `<email>` with the user email. The placeholder `<dev-url>`
-  should point to your development environment URL.
+  In the example replace the placeholder `<api-key>` with an existing API
+  key and placeholder `<email>` with the bot's email. The placeholder 
+  `<dev-url>`should point to your development environment URL.
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: Please add space between backtick and `should`

Comment (trueskawka) [inline tasks/interactive-bots.md]: I think reversing this sentence makes it confusing - the intention was "replace the <api-key> provided in the example with the API key in your file", but the rephrasing sounds "replace the <api-key> in the example".

File: tasks/interactive-bots.md [hunk 2]
```diff
@@ -110,8 +110,8 @@ doing this setup process. Add your notes as a `notes.md` file to
 * Create a commit with the screenshots and notes, with commit message
 `interactive bots: Run the followup bot for <username>.`.
 
-* Create a pull request in the `zulip/zulip-gci` repository, with title
-`interactive bots: Run the followup bot for <username>`.
+* Create a pull request in the (`zulip/zulip-gci`)[https://github.com/zulip/zulip-gci] repository, with title
```
Comment (trueskawka) [inline tasks/interactive-bots.md]: I don't think links are necessary here - we would have to change it in all the tasks to be consistent and I'd think a GCI contestant knows which repository we mean from the beginner tasks.

File: tasks/interactive-bots.md [hunk 3]
```diff
@@ -110,8 +110,8 @@ doing this setup process. Add your notes as a `notes.md` file to
...
+`interactive bots: Run the followup bot for <username>`. Link to your GCI task in comment section.
```
Comment (trueskawka) [inline tasks/interactive-bots.md]: Could you add it for all other tasks in this file?

**Comment by trueskawka** [comment]
Hi @KaiCode2! 

I've added some additional comments, please take them into account when working on this.

File: tasks/interactive-bots.md (hunk 1 above)
Comment (KaiCode2) [inline tasks/interactive-bots.md]: @trueskawka Do you feel that it's worth reverting back to the old wording?

**Comment by femnad** [comment]
Hey,

Could you change you commit subjects to start with `interactive bots:`. Also I think you can only have one comment here since you're only doing changes on one file for the same task/goal.

**Comment by KaiCode2** [comment]
@femnad Squashed commits and renamed to start with `interactive bots:`. Let me know if everything looks ok 😄

**Comment by umairwaheed** [comment]
LGMT.

**Pull request merged**